test/Chirp.CLI.Client.Tests/UnitTests.cs

```diff
@@ -1,10 +1,23 @@
+using System.Text.RegularExpressions;
+using System.Xml.XPath;
+using Chirp.CLI.Client;
+using Xunit;
+
 namespace Chirp.CLI.Client.Tests;
 
 public class Tests
 {
+    //UNITTEST -> Test that the UNIX time is converted to a string correctly
     [Test]
-    public void Test1()
+    public void TestUnixConversion()
     {
-        Assert.Pass();
+        //Arrange
```

Comment:
Unnecessary comment, we will likely have many more tests in there in the future. We don't want every 3rd line to be a comment, unless there is absolute doubt as to what is happening. For lines 14, 17 and 20, replace the 3 comments with 1 where it is most needed, if any at all. Again, there shouldn't be much confusion with simple tests as this.